api/src/test/java/uk/gov/dhsc/htbhf/claimant/message/processor/DetermineEntitlementMessageProcessorTest.java

```diff
@@ -94,17 +109,75 @@ void shouldSuccessfullyProcessMessageAndTriggerPaymentWhenClaimantIsEligible() {
         verify(paymentCycleService).updatePaymentCycle(context.getCurrentPaymentCycle(), decision);
         MessagePayload expectedPayload = MessagePayloadFactory.buildMakePaymentMessagePayload(context.getCurrentPaymentCycle());
         verify(messageQueueClient).sendMessage(expectedPayload, MessageType.MAKE_PAYMENT);
-        verifyZeroInteractions(claimRepository, pregnancyEntitlementCalculator);
+        verifyZeroInteractions(claimRepository, pregnancyEntitlementCalculator, childDateOfBirthCalculator, childDateOfBirthCalculator);
+    }
+
+    @ParameterizedTest(name = "Qualifying benefit status={0}")
+    @ValueSource(strings = {"CONFIRMED", "NOT_CONFIRMED"})
+    void shouldExpireClaimAndSendNoLongerOnSchemeEmailWhenEligibleWithNoChildrenAndNotPregnant(
```

Comment:
I don't understand how someone could be eligible when their `QualifyingBenefitEligibilityStatus` is `NOT_CONFIRMED`? There's also a comment to the same effect on line 138